prime-router/docs/api/lookup-tables-openapi.yaml

```diff
@@ -106,6 +106,38 @@ paths:
               schema:
                 $ref: '#/components/schemas/ErrorMsg'
 
+  /api/lookuptables/{tableName}/content:
```

Comment:
This is the new endpoint that gets table content of the active version of the table.